index.html

```diff
@@ -6,10 +6,10 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>BC Business Registry</title>
     <link rel="icon" href="dummy-not-used-do-not-remove">
-    <link rel="icon" href="<%= BASE_URL %>favicon.png">
-    <link rel="preload" href="<%= BASE_URL %>fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
-    <link rel="stylesheet" href="<%= BASE_URL %>css/addresscomplete-2.30.min.css">
-    <script type="text/javascript" src="<%= BASE_URL %>js/addresscomplete-2.30.min.js" defer></script>
+    <link rel="icon" href="/favicon.png">
+    <link rel="preload" href="/fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
I wonder if we should preload some fonts in Edit UI and Create UI?

Reply:
As far as I know, I don't think so Sev?

Reply:
I remember playing with this (years ago), and this line preloads the MDI icons so the page can display then when it first loads -- probably for the initial spinner.

But we don't want to go crazy with preloading fonts as that delays the first render (ie, performance).

If Edit UI and Create UI don't show a spinner initially then they won't benefit from this. Otherwise, they might.

Reply:
Future problem. Don't worry about it now.